sql/expression/function/aggregation/window_framer.go

```diff
@@ -258,6 +262,17 @@ func (f *GroupByFramer) SlidingInterval(ctx sql.Context) (sql.WindowInterval, sq
 	panic("implement me")
 }
 
+// rowFramerBase is a sql.WindowFramer iterator that tracks
+// index frames in a sql.WindowBuffer using integer offsets.
+// Only a subset of bound conditions will be set for a given
+// framer implementation, one start and one end bound.
+//
+// Ex: startCurrentRow = true; endNFollowing = 1;
+//     buffer = [0, 1, 2, 3, 4, 5];
+// =>
+// pos:    0->0   1->1   2->2   3->3   4->4   5->5
+// frame:  {0,2}, {1,3}, {2,4}, {3,5}, {4,6}, {4,5}
+// rows:   [0,1], [1,2], [2,3], [3,4], [4,5], [5]
 type rowFramerBase struct {
```

Comment:
Comment is good but this struct still has a lot of optional values where the defaults aren't clear. Make them explicit.